components/GenericList.js

```diff
@@ -0,0 +1,18 @@
+const GenericList = ({ items, value }) => {
```

Comment:
@easherma There is really no reason to use this. It's an anti-pattern trying to reimplement native JS functionality.